packages/react/src/components/Attachment/Attachment.tsx

```diff
@@ -136,8 +136,10 @@ class Attachment extends UIComponent<WithAsProp<AttachmentProps>, AttachmentStat
   }
 
   performClick = e => {
-    e.stopPropagation()
-    this.handleClick(e)
+    if (e.currentTarget === e.target) {
```

Comment:
based on agreement with @layershifter and @miroslavstastny this is ok for now, but it shows a bigger issue we have related to click handling of components that have parent/child clickable elements, especially if the elements trigger click on keyboard events (button, a)

This might help move the logic to behaviors: https://github.com/stardust-ui/react/issues/1502